### ISSUE TITLE
Ensure that index.html is valid according to the w3c validator

### ISSUE THREAD

**Issue by rreitmann** [dzhw/metadatamanagement#1064]
In order to improve accessibility we need to ensure that the index.html passes the w3c validator without configuring any exceptions. This should be implemented as grunt task...
